static-analysis.sh

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
```

Comment:
Let's move this to the tools directory.

Reply:
I didn't do that initially because `tools` looks like a go package, so didn't wanna mix things by adding a shell script in there, but I'm not particularly strict about it so I guess it's fine